pkgs/applications/misc/tandoor-recipes/common.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub }:
+let
+  version = "1.3.3";
+  srcSha256 = "sha256-hnTDwnVTvVxSVzwUYBHBJpMqsvl/o011nkddYSJQQdQ=";
+  yarnSha256 = "sha256-HnFCz+okFqLddxyQ56NPVmW6Z/mFqVe/7Ihp+Mol12I=";
+in
+{
+  inherit version yarnSha256;
+
+  src = fetchFromGitHub {
+    owner = "TandoorRecipes";
+    repo = "recipes";
+    rev = version;
+    sha256 = srcSha256;
+  };
+
```

Comment:
```suggestion

{
  version = "1.3.3";

  src = fetchFromGitHub {
    owner = "TandoorRecipes";
    repo = "recipes";
    rev = version;
    sha256 = "sha256-hnTDwnVTvVxSVzwUYBHBJpMqsvl/o011nkddYSJQQdQ=";
  };

  yarnSha256 = "sha256-HnFCz+okFqLddxyQ56NPVmW6Z/mFqVe/7Ihp+Mol12I=";
```

Reply:
I kept them in the `let ... in` block to group together the lines that would be modified by the update script.

I think it is neater that way.

Reply:
using `rev = version` requires the set to be `rec` or using new function argument.

Not sure what to prefer, though introducing a `let` just to "groups" things is not really something we do often.

Also when looking at a derivation, I hate when I do have to search for the hashes *somewhere* rather than directly under the key.

So I totally second Sandros request

Reply:
True, I do agree that conforming to what the rest of nixpkgs does has value.

Will do that appropriate changes.